curriculum/challenges/english/02-javascript-algorithms-and-data-structures/regular-expressions/find-characters-with-lazy-matching.english.md

```diff
@@ -26,7 +26,10 @@ Fix the regex <code>/&lt;.*&gt;/</code> to return the HTML tag <code>&lt;h1&gt;<
 tests:
   - text: The <code>result</code> variable should be an array with <code>&lt;h1&gt;</code> in it
     testString: assert(result[0] == '<h1>');
-
+  - text: <code>myRegex</code> should use lazy matching
+    testString: assert(myRegex.test(myRegex.source))
+  - text: The <code>result</code> variable should not match the string 'h1'
+    testString: assert(result[0] !== 'h1')
```

Comment:
```suggestion
    testString: assert(!myRegex.source.match('h1'))
```
If the first test passes, we already know that the result is not 'h1', so this test should check the source.